src/app/store.ts

```diff
@@ -0,0 +1,10 @@
+import {combineReducers, createStore} from 'redux'
+import hotelReducer from '../hotel/redux/hotelReducer'
+
+const store = createStore(combineReducers({
```

Comment:
It's better to create a folder with the name 'store' and then create there folders like hotels, common and etc

Reply:
It's better to create a folder with the name 'store' and then create there folders like hotels, common and etc